delivery.yaml

```diff
@@ -0,0 +1,62 @@
+build_steps:
+    - desc: 'Install Docker'
+      cmd: |
+        # enable experimental features (--squash)
+        mkdir -p /etc/systemd/system/docker.service.d
+        cat << EOF > /etc/systemd/system/docker.service.d/docker.conf
+        [Service]
+        ExecStart=
```

Comment:
I copied this shamelessly from Henning 😆 
E.g. here https://github.com/zalando-stups/docker-ubuntu/blob/master/delivery.yaml